Buscorepuestos/buscorepuestos-web
# Feature/br 354 card info

## What type of PR is this? (check all applicable)


## Description
The category card component was redesigned and made responsive. Similarly, the width of the div that wraps the component is under review.


## Related Tickets & Documents
https://linear.app/buscorepuestos/issue/BR-354/card-info

## QA Instructions, Screenshots, Recordings

Verify card category component

Also make sure you check this browser compatibility version for desktop and mobile:

## Checklist

DO NOT send the PR without completing the checklist above:


## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## Are there any post deployment tasks we need to perform?
